includes/emails/template.php

```diff
@@ -78,6 +78,8 @@ function give_email_preview_template_tags( $message ) {
 	$message = str_replace( '{name}', $user->display_name, $message );
 	$message = str_replace( '{fullname}', $user->display_name, $message );
 	$message = str_replace( '{username}', $user->user_login, $message );
+	$message = str_replace( '{user_email}', $user->user_email, $message );
+	$message = str_replace( '{billing_address}', '', $message );
```

Comment:
Just use a fake address here like:

 > 123 Test Street, Unit 222
> Somewhere Town, CA, 92101

If it's blank that will confuse people.